### ISSUE TITLE
Graph column not showing when I use dfSummary on macbook pro

### ISSUE THREAD

**Issue by Tolu238** [dcomtois/summarytools#89]
Thanks for creating this fantastic package. I find that when I use dfSummary on a macbook pro, the graph column is excluded unlike on a PC. Have you received any such reports too? I am in transit now or I could have provided a screenshot. Do you have any suggestions for getting around this?

**Comment by Tolu238** [comment]
<img width="1595" alt="Screen Shot 2019-09-13 at 3 36 47 AM" src="https://user-images.githubusercontent.com/13739201/64849048-2a3ffe80-d5d8-11e9-9338-90444a722051.png">

Screenshot attached. 
XQuartz installed.
Will try uninstalling summarytools and then installing the development version.

**Comment by Tolu238** [comment]
<img width="1622" alt="Screen Shot 2019-09-13 at 3 58 36 AM" src="https://user-images.githubusercontent.com/13739201/64850614-5315c300-d5db-11e9-9508-f495e23a9322.png">

Thanks once again. I think I got it sorted.
I uninstalled knitr, pander, and rmarkdown (saw you suggested that elsewhere).
Then followed your suggestions here, (step 3 and then 1 (with ref = "dev-current")): https://stackoverflow.com/questions/49604912/r-error-when-installing-packages-summarytools
